docs/test/reference-qa.md

```diff
@@ -560,6 +560,39 @@ For more details, see [Add findings to existing bugs with exploratory testing](a
 2. Search for 'Use Blink's zoom for device scale factor'
 3. Change it to **Disabled**
 
+## FAQ on New Test Plans Page
+
+### Q: How can I assign configurations to my test cases?
+**A:** You can assign configurations in two ways: 
+
+a. To assign configuration to whole suite, go to test suite tree, select the test suite and open the in-context menu bar to assign configurations.
+
```

Comment:
The image FAQ 1.1 needs to be added here. Along with this remove the FAQ 1.1 image since we don't want to promote that way actively. It being in text is fine.

Reply:
Done